Project 14/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<head>
+	<title>Grocery List</title>
+	<link rel="stylesheet" href="index.css">
+</head>
+<body>
+	<div id="wrapper" class="wrapper">
+		<h1 class="header">Grocery List</h1>
+		<form action="#" onsubmit="app.addItem(); return false;">
+			<label for="input-field"></label>
+			<input id="input-field" class="input-field" placeholder="Enter Value ..." required>
+			<button type="submit" class="button">New Item</button>
```

Comment:
```suggestion
			<button type="submit" class="button">Add item</button>
```
Кнопки называются либо глаголом, либо глагол + сабжект к которому относится действие